manifests/m/MichaelDiCarlo/PostyBirb/3.1.58/MichaelDiCarlo.PostyBirb.installer.yaml

```diff
@@ -0,0 +1,18 @@
+# Created with WinGet Updater using komac v2.11.2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: MichaelDiCarlo.PostyBirb
+PackageVersion: 3.1.58
+InstallerLocale: en-US
+InstallerType: nullsoft
+ProductCode: caa559a3-e745-546c-923c-94a7ab26447f
+ReleaseDate: 2025-04-02
+AppsAndFeaturesEntries:
+- DisplayName: PostyBirb+ 3.1.58
+  ProductCode: caa559a3-e745-546c-923c-94a7ab26447f
+Installers:
+- Architecture: x86

```

Comment:
```suggestion
- Architecture: x64
```
Try x64?

Reply:
It installs fine on x64:
<img width="1728" alt="Screenshot 2025-04-05 at 3 05 39 PM" src="https://github.com/user-attachments/assets/8bf562ab-d135-473d-8e21-3b7acba08fb3" />